src/js/background/background.js

```diff
@@ -162,11 +162,13 @@ async function refreshAccountPages() {
   if (!settingsRefresh) {
     browser.storage.local.set({ settingsRefresh: true });
 
-    browser.tabs.query({ url: "http://127.0.0.1/*" }, function (tabs) {
+    function tabReloader(tabs) {
       for (let tab of tabs) {
         browser.tabs.reload(tab.id);
       }
-    });
+    };
+
+    browser.tabs.query({url: "http://127.0.0.1/*"}).then(tabReloader);   
```

Comment:
This was a browser quirk for Chrome – no looped async callbacks. I had to move it a `.then` function.